modules/monitoring/application/forms/Config/TransportConfigForm.php

```diff
@@ -351,9 +354,12 @@ public function isValid($formData)
             return false;
         }
 
-        if (! ($this->getElement('transport_validation') === null || (
-            $this->isSubmitted() && isset($formData['force_creation']) && $formData['force_creation'])
-        )) {
+        if ($this->getSubForm('transport_form') instanceof ApiTransportForm) {
+            if (isset($formData['force_creation']) && $formData['force_creation']) {
+                // ignore any validation result
+                return true;
+            }
+
             try {
                 CommandTransport::createTransport(new ConfigObject($this->getValues()))->probe();
```

Comment:
Please only do this if the transport has such a method. If not, just return true as if the test succeeded.